frontend/views/containers/access/SignupModal.vue

```diff
@@ -28,10 +28,7 @@ export default ({
   methods: {
     submit () {
       if (this.$route.query.next) {
-        // TODO: get rid of this timeout and fix/update tests accordingly
-        setTimeout(() => {
-          this.$router.push({ path: this.$route.query.next })
-        }, 1000)
+        this.$router.push({ path: this.$route.query.next }).catch(logExceptNavigationDuplicated)
```

Comment:
I'm not a fan of `logExceptNavigationDuplicated` becoming commonplace. Although it's arguably convenient, I'd rather have an if-statement upfront to prevent the duplicate navigation.

Reply:
An `if` statement would turn what should be a 1-liner into 3 lines :-\